cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaAvailability.java

```diff
@@ -55,17 +61,17 @@
      * producers with acks=all publishing to topics with a {@code min.in.sync.replicas}.
      */
     Future<Boolean> canRoll(int podId) {
-        log.debug("Determining whether broker {} can be rolled", podId);
+        log.debug("{}: Determining whether broker {} can be rolled", reconciliation, podId);
         return canRollBroker(descriptions, podId);
     }
 
     private Future<Boolean> canRollBroker(Future<Collection<TopicDescription>> descriptions, int podId) {
         Future<Set<TopicDescription>> topicsOnGivenBroker = descriptions
                 .compose(topicDescriptions -> {
-                    log.debug("Got {} topic descriptions", topicDescriptions.size());
+                    log.debug("{}: Got {} topic descriptions", reconciliation, topicDescriptions.size());
                     return Future.succeededFuture(groupTopicsByBroker(topicDescriptions, podId));
                 }).recover(error -> {
-                    log.warn(error);
+                    log.warn(new ParameterizedMessage("{}: failed to get topic descriptions", reconciliation), error);
```

Comment:
Why not just `log.warn("{}: failed to get topic descriptions", reconciliation, error)`? (A "spare" exception argument, for which there is no `{}`, still gets dumped to the log).

Reply:
Oh sweet, that's way better - I was wondering why my IDE hadn't highlighted it as a warning, and thought it was just an IDE issue.

Reply:
Changes made, thanks @tombentley